app/assets/javascripts/modal/modal-workflow.js

```diff
@@ -34,6 +35,14 @@ ModalWorkflow.prototype.renderError = function (result) {
   window.Raven.captureException(result)
   console.error(result)
   this.$multiSectionViewer.showStaticSection('error')
+  this.setModalLabel()
```

Comment:
Looks like we're missing a corresponding test for the 'render error aria-label' state.